kimchi/js/bindings.js

```diff
@@ -1866,8 +1857,154 @@ function linearization_of_rust(linearization, affine_class) {
 // Provides: None
 var None = 0;
 
+// Provides: caml_is_none
+// Requires: None
+var caml_is_none = function (v) {
+  return v === None;
+};
+
+// Provides: caml_map_of_rust_vector
+// Requires: js_class_vector_of_rust_vector
+var caml_map_of_rust_vector = function (v, klass, converter_to_rust) {
+  var a = js_class_vector_of_rust_vector(v, klass);
+  var res = [0];
+  for (var i = 0; i < a.length; ++i) {
+    // TODO Check this. Could be off by 1
+    res.push(converter_to_rust_rust(a[i]));
+  }
+  return res;
+};
+
+// Provides: caml_opt_of_rust
+// Requires: None
+var caml_opt_of_rust = function (value, value_of_rust) {
+  if (value == undefined) {
```

Comment:
maybe `===`? It is semantically the same for `undefined`, I think.

Reply:
Fixed in [87b72e7](https://github.com/o1-labs/snarkyjs-bindings/pull/114/commits/87b72e74a3688e1d2734d28b73b0f48199d87897)